MerlinAU.sh

```diff
@@ -4591,17 +4595,17 @@ _GetOfflineFirmwareVersion_()
     export release_version="$formatted_version"
 }
 
-##----------------------------------------##
-## Modified by Martinski W. [2024-Jul-30] ##
-##----------------------------------------##
-_SelectOfflineZipFile_()
+##------------------------------------------##
+## Modified by ExtremeFiretop [2024-Aug-06] ##
+##------------------------------------------##
+_SelectOfflineUpdateFile_()
 {
-    local selection  zipFileList  fileCount
+    local selection fileList fileCount
 
-    # Check if the directory is empty or no ZIP files are found #
-    if [ -z "$(ls -A "$FW_ZIP_DIR"/*.zip 2>/dev/null)" ]
+    # Check if the directory is empty or no desired files are found #
+    if [ -z "$(ls -A "$FW_ZIP_DIR"/*.w "$FW_ZIP_DIR"/*.pkgtb "$FW_ZIP_DIR"/*.zip 2>/dev/null)" ]
```

Comment:
You want to accept raw image files as well as ZIP files?
That means the SHA256 signature must still be found as a separate file along with the image, correct?

Reply:
The raw file is only for Gnuton support.

The instructions will say a ZIP, if they drop a raw file it won't work since it's expecting a zip for the rest of the update to work. (It would just fail at the next step further down).

But as I touched on, this is kinda a pre-sync step, since I do want to accept raw files for Gnuton.

Reply:
> The raw file is only for Gnuton support.
> 
> The instructions will say a ZIP, if they drop a raw file it won't work since it's expecting a zip for the rest of the update to work. (It would just fail at the next step further down).
> 
> But as I touched on, this is kinda a pre-sync step, since I do want to accept raw files for Gnuton.
>
I thought Gnuton F/W updates were packaged in ZIP files just like RMerlin's are. Honestly, I haven't really looked closely at Gnuton's so I just assumed that the packaging method was the same.

Reply:
@Martinski4GitHub 

I wish! That would of made our Gnuton PR WAY smaller hahhaha